models/forced_lorenz_96/README.rst

```diff
@@ -0,0 +1,217 @@
+#######################
+Forced Lorenz 96 README
+#######################
+
+Contents
+========
+
+#. `Overview`_
+#. `Quick Start`_
+#. `Namelist`_
+#. `Terms of Use`_
+#. `References`_
+
+Overview
+========
+
+The *forced_lorenz_96* model implements the standard Lorenz (1996) [1]_
+equations except that the forcing term, F, is added to the state vector and is
+assigned an independent value at each gridpoint. The result is a model
+that is twice as big as the standard L96 model. The forcing can be
+allowed to vary in time or can be held fixed so that the model looks
+like the standard L96 but with a state vector that includes the constant
+forcing term. An option is also included to add random noise to the
+forcing terms as part of the time tendency computation which can help in
+assimilation performance. If the random noise option is turned off (see
+namelist) the time tendency of the forcing terms is 0.
+
+DART state vector composition:
+
+  **state variables** - *traditional Lorenz_96 state*
+  indices 1 - 40
+
+  **forcing terms** - *"extended" state*
+  indices 41 - 80
+
+The DART tutorial Section 20 explores parameter estimation using the
+*forced_lorenz_96* model.
+
+Quick Start
+===========
+
+To become familiar with the model, try this quick experiment.
+
+#. compile everything in the *model/forced_lorenz_96/work* directory.
+
+   .. container:: unix
+
+      cd models/forced_lorenz_96/work
+      ./quickbuild.csh
```

Comment:
Somehow this rendered into a single line

Reply:
Fixed.